lib/widgets/tripStepSelectBox_widget.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:gra_terenowa/controller/app_controller.dart';
+import 'package:gra_terenowa/controller/tripTab_controller.dart';
+import 'package:gra_terenowa/extras/colors.dart';
+import 'package:gra_terenowa/extras/statics.dart';
+
+class TripStepSelectBox extends StatelessWidget {
```

Comment:
There is an issue with box overflow. Maybe we could make this select box scrollable to fit smaller screen sizes.
![image](https://user-images.githubusercontent.com/47281254/147741590-9e255c0f-5118-480e-adf4-d6f986540962.png)

Reply:
fixed